app/editor/src/features/content/combined-view/CombinedView.tsx

```diff
@@ -0,0 +1,59 @@
+import 'react-reflex/styles.css';
```

Comment:
Initially had logic on regular content form for a condensed version for the half screen size but got too messy. Made this instead. I think spending some time on the scalability of our components would be beneficial as they "kinda" are right now. But they do not look great when the collapse.